coduno.go

```diff
@@ -96,11 +104,11 @@ func setupHandler(handler Handler) http.HandlerFunc {
 			cors(w, r)
 			return
 		}
-		session, _ := cs.Get(r, sessionID)
-		if session.IsNew {
-			http.Error(w, "Unauthorized session.", http.StatusUnauthorized)
-			return
-		}
+		// session, _ := cs.Get(r, sessionID)
+		// if session.IsNew {
+		// 	http.Error(w, "Unauthorized session.", http.StatusUnauthorized)
+		// 	return
+		// }
 		ctx := setupBaseHandler(w, r)
```

Comment:
considering the security will be changed, this makes it easier to develop locally untill that happens